alfred-telemetry-solr/README.md

```diff
@@ -80,23 +80,23 @@ At the moment following metrics are included, for each core tracked by solr:
 
 | Name                                                                   |
 | -----------------------------------------------------------------------|
-| alfresco_nodes{core=<core>,feature="Alfresco Nodes in Index"}          |
-| alfresco_nodes{core=<core>,feature="Alfresco Transactions in Index"}   |
-| alfresco_nodes{core=<core>,feature="Alfresco Stated in Index"}         |
-| alfresco_nodes{core=<core>,feature="Alfresco Unindexed Nodes"}         |
-| alfresco_nodes{core=<core>,feature="Alfresco Error Nodes in Index"}    |
-| alfresco_acls{core=<core>,feature="Alfresco Acls in Index"}            |
-| alfresco_acls{core=<core>,feature="Alfresco Acl Transactions in Index"}|
+| alfresco.nodes{core=<core>,feature="Alfresco Nodes in Index"}          |
+| alfresco.nodes{core=<core>,feature="Alfresco Transactions in Index"}   |
```

Comment:
shouldn't this be something like `alfresco.transactions` ?

Reply:
My thinking about this structure was: 

- there are 2 trackers (for nodes and for acls) => alfresco_nodes and alfresco_acls
- I took features from the SUMMARY as they are
- in grafana they are shown in different rows and panels, based on semantic distinctions

Splitting further can be done, with consequences that grafana panels need to be changed as well and the code need to deal separately with each feature.

Reply:
In my opinion, that's worth it.

Reply:
I pushed a proposal.